### ISSUE TITLE
word2vec.py - reset_from bug

### ISSUE THREAD

**Issue by bruderjakob12** [RaRe-Technologies/gensim#1268]
#### Description
gensim.models.word2vec: reset_from seems to use the wrong variables for vocab and index2word

#### Steps/Code/Corpus to Reproduce
```
from gensim.models import word2vec

sentences = ['human', 'machine']
model = word2vec.Word2Vec()
model.build_vocab(sentences)
new_model = word2vec.Word2Vec()
new_model.reset_from(model)
```
#### Expected Results

new_model initialized with settings from model

#### Actual Results

Traceback (most recent call last):
  File "test.py", line 8, in <module>
    new_model.reset_from(model)
  File "/usr/local/lib/python2.7/dist-packages/gensim/models/word2vec.py", line 734, in reset_from
    self.wv.vocab = other_model.vocab
AttributeError: 'Word2Vec' object has no attribute 'vocab'

#### Versions

gensim 1.0.1

#### Fix

gensim/models/word2vec.py

Line 734 change 
self.wv.vocab = other_model.vocab
to
self.wv.vocab = other_model.wv.vocab

Line 735 change
self.wv.index2word = other_model.index2word
to
self.wv.index2word = other_model.wv.index2word

**Comment by piskvorky** [comment]
@tmylk needs unit test.

**Comment by bruderjakob12** [comment]
I guess it is fixed by #1230
